src/telemetry/metrics.yaml

```diff
@@ -1776,3 +1697,240 @@ session:
+      check did not return, and milliseconds later that network check returns and another health
+      check is started because of the return.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-5860
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/9057#pullrequestreview-1868746289
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - mcleinman@mozilla.com
+      - vpn-telemetry@mozilla.com
+    expires: 2024-07-15
+
+  unstable_count:
+    type: counter
+    lifetime: ping
+    send_in_pings:
+      - vpnsession
+      - daemonsession
+    description: |
+      Count of times that the connection health check results
+      in unstable.
+
+      The health check counters must not be considered as markers of time. There is a possible
+      situation in the health check (which calls the telemetry) which frequently results in more
+      than one count per second. The situation: A health check is conducted because a network
+      check did not return, and milliseconds later that network check returns and another health
+      check is started because of the return.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-5860
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/9057#pullrequestreview-1868746289
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - mcleinman@mozilla.com
+      - vpn-telemetry@mozilla.com
+    expires: 2024-07-15
+
+  stable_count:
+    type: counter
+    lifetime: ping
+    send_in_pings:
+      - vpnsession
+      - daemonsession
+    description: |
+      Count of times that the connection health check succeeds.
+
+      The health check counters must not be considered as markers of time. There is a possible
+      situation in the health check (which calls the telemetry) which frequently results in more
+      than one count per second. The situation: A health check is conducted because a network
+      check did not return, and milliseconds later that network check returns and another health
+      check is started because of the return.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-4700
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/6783#pullrequestreview-1407551157
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/9057#pullrequestreview-1868746289
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - mcleinman@mozilla.com
+      - vpn-telemetry@mozilla.com
+    expires: 2024-07-15
+
+  no_signal_time:
+    type: timing_distribution
+    lifetime: ping
+    send_in_pings:
+      - vpnsession
+      - daemonsession
+    description: |
+      Time spent in no signal state.
+
+      Only collected on desktop for vpnsession, as mobile apps
+      frequently are relaunched during VPN sessions. It is
+      collected in daemonsession, currently only on Android.
```

Comment:
If there are plans to add this to iOS it would be worth adding a ticket reference here.

Reply:
This applies to all comments where we say "currently only on Android" or similar.

Reply:
Added ticket in there